lib/parser.js

```diff
@@ -145,7 +145,7 @@ module.exports = class Parser {
         seg.push(tok);
         this._t.next();
       } else if (isList(tok)) {
-        if (inList && this._t.previous.name === 'linebreak') {
```

Comment:
Just FYI, this isn't a directly related change, but this check was no longer necessary a while back since the lexer handles this case now.